### PR TITLE
Feature/mutagen script resource path

### PR DESCRIPTION
**Pull request by jcwuzoegiver**
Hi @iizrailevsky @jwtodd,

Can you pls review this PR? This is for allow the mutagen scripts root resource path to be configurable.

Thanks,
Jason

### CODE REVIEW

**Comment by iizrailevsky** [comment]
Nicely done @jcwuzoegiver! Thanks for taking care of unit tests as well. :shipit:

File: modules/repository/src/main/java/com/intuit/wasabi/repository/RepositoryModule.java [hunk 1]
```diff
@@ -69,6 +69,10 @@ protected void configure() {
                 .toInstance(Boolean.valueOf(getProperty("assign.user.to.new", properties, TRUE.toString())));
         bind(String.class).annotatedWith(named("default.time.format"))
                 .toInstance(getProperty("default.time.format", properties, "yyyy-MM-dd HH:mm:ss"));
+        bind(String.class).annotatedWith(named("cassandra.mutagen.root.resource.path"))
```
Comment (jwtodd) [inline modules/repository/src/main/java/com/intuit/wasabi/repository/RepositoryModule.java]: beautiful

File: modules/repository/src/main/java/com/intuit/wasabi/repository/impl/cassandra/CassandraExperimentRepositoryModule.java [hunk 2]
```diff
@@ -74,14 +75,14 @@ protected CassandraDriver provideExperimentDriver(HealthCheckRegistry registry)
...
     protected ExperimentRepository provideExperimentRepository(@ExperimentDriver CassandraDriver driver,
                                                                ExperimentsKeyspace keyspace,
-                                                               ExperimentValidator validator)
+                                                               ExperimentValidator validator,
+                                                               final @Named("cassandra.mutagen.root.resource.path") String mutagenRootResourcePath)
             throws IOException, ConnectionException {
         LOGGER.debug("Providing Cassandra experiment repository instance");
 
         CassandraMutagen mutagen = new CassandraMutagenImpl();
-        String rootResourcePath = "com/intuit/wasabi/repository/impl/cassandra/experiments/mutation";
```
Comment (jwtodd) [inline modules/repository/src/main/java/com/intuit/wasabi/repository/impl/cassandra/CassandraExperimentRepositoryModule.java]: purge hardwires ftw

File: modules/repository/src/test/java/com/intuit/wasabi/repository/impl/database/DatabaseAnalyticsTest.java [hunk 3]
```diff
@@ -51,13 +51,13 @@
     public void setup() throws SQLException {
         when(transactionFactory.newTransaction()).thenReturn(transaction);
         when(transactionFactory.getDataSource()).thenReturn(dataSource);
-        databaseAnalytics = spy(new DatabaseAnalytics(transactionFactory, flyway));
+        databaseAnalytics = spy(new DatabaseAnalytics(transactionFactory, flyway, "com/intuit/wasabi/repository/impl/mysql/migration"));
```
Comment (jwtodd) [inline modules/repository/src/test/java/com/intuit/wasabi/repository/impl/database/DatabaseAnalyticsTest.java]: nit: for unit tests, i like using fake/mock data where possible

not sure if this is a pure unit test? or an integration test pretending to be a unit test?

not a blocker either way

**Comment by jwtodd** [comment]
:shipit:

**Comment by jwtodd** [comment]
:shipit:

**Pull request merged**